nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -545,4 +545,81 @@ def export_model(self, model_path, calibration_path=None, onnx_path=None, input_
 
         self.export_model_save(self.bound_model, model_path, calibration_config, calibration_path, onnx_path, input_shape, device)
 
-        return calibration_config
\ No newline at end of file
+        return calibration_config
+
+
+class LsqQuantizer(Quantizer):
```

Comment:
Please add docstring as the other Quantizers.

Reply:
done